ResearchKit/Common/ORKCountdownLabel.m

```diff
@@ -34,8 +34,8 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 @interface ORKCountdownLabel ()
 
 
-@property (nonatomic, strong) NSString *mmString;
-@property (nonatomic, strong) NSString *ssString;
+@property (nonatomic, copy) NSString *mmString;
+@property (nonatomic, copy) NSString *ssString;
```

Comment:
These are fine, although technically irrelevant, since the setters are never called on these internal properties.